openmdao/docs/other/om_command.rst

```diff
@@ -7,38 +7,42 @@ Command Line Tools
 OpenMDAO has a number of debugging/viewing command line tools that are available via the `openmdao`
 command.  There are two types of commands available, those that perform some sort of viewing or
 configuration checking on the Problem after its setup is complete, and those that are used to
-collect information about the entire run of the Problem, things like profilers and tracers.
+collect information about the entire run of the Problem, like profilers and tracers.
 
 .. note::
     The `openmdao` sub-commands, as well as any other console scripts associated with OpenMDAO, will
     only be available if you have installed OpenMDAO using *pip*. See :ref:`Getting Started <GettingStarted>`
 
 
+.. note::
+    When using a command line tool on a script that takes its own command line arguments, those
+    arguments must be placed after a :code:`--` on the command line.  Anything to the right of the
+    :code:`--` will be ignored by the openmdao command line parser and passed on to the user script.
```

Comment:
Kind of feel like an example would help here. (even just a syntax example like the one in the help text, not something that will run in doctest)

Reply:
fixed